api/Request.json

```diff
@@ -496,10 +496,10 @@
               "version_added": "47"
             },
             "safari": {
-              "version_added": "11"
```

Comment:
This "11" came from https://github.com/mdn/browser-compat-data/pull/937 but I see no discussion of it so am guessing it was a mistake.